user/insert.sql

```diff
@@ -0,0 +1 @@
+INSERT INTO users (id, username, email) VALUES ('1234', 'yatharth_mathur', 'yatharth.mathur@gmail.com');
```

Comment:
capitalize syntax and add user first name

Reply:
added a fix 8e4dbe